extscheduler/resources/schemas/dbscripts/sqlserver/extscheduler-21.001-21.002.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE extscheduler.Resources ADD  Instance varchar(50);
```

Comment:
Please add this new column to the extscheduler.xml file too. This will keep our automated tests passing on TeamCity.

Reply:
The field "Instance" is added to the xml file.

Reply:
I see Build #144 failed with a gradle exception. Did I miss anything?
https://teamcity.labkey.org/buildConfiguration/LabKey_2011Release_External_Onprc_OnprcInstaller?mode=builds

Reply:
No, this failure is because we're doing an upgrade for a server that TeamCity uses:

https://www.labkey.org/Premium/announcements-thread.view?rowId=21295

Your build snuck in before the cutoff, but wasn't done in time to avoid the error. The maintenance should be done soon, and we can re-queue an installer build then.